engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -193,7 +198,8 @@ public void set(String value) {
             } catch (UnresolvedWorldGeneratorException e) {
                 logger.error("Can't configure the world! due to {}", e.getMessage());
             }
-        });
+        }
```

Comment:
No need to split this across two-lines. The one-line approach matches the current style. Might be some sort of auto-formatting in your IDE?